scripts/main.js

```diff
@@ -71,4 +71,60 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+console.log(" De afdeling Sale heeft " + departments.sales.numberOfEmployees + "medewerkers");
+console.log("Marketing is een leuke afdeling om te werken" + departments.marketing.description + ".");
+console.log("De afdeling Customer service heeft" +departments["customer-service"].numberOfEmployees +" medewerkers.");
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager." + departments.sales.jobs[1].description);
+const userInput = prompt("Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]");
+// const userInput = prompt("Je koos marketing. Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in. \n 0 = "
+//     + departments.marketing.jobs[0].title + "\n 1 = " + departments.marketing.jobs[1].title + "\n 2 = " + departments.marketing.jobs[2].title + "\n 3 = "
+//     + departments.marketing.jobs[3].title);
+
+console.log(userInput.toLowerCase());
+switch (userInput.toLowerCase()) {
+    case "marketing":
+        console.log("Je koos " + userInput.toLowerCase() + ". " + departments[userInput].description);
+        break;
+    case "sales":
+        console.log("Je koos " + userInput.toLowerCase() + ". " + departments[userInput].description);
+        break;
+    case "customer-service":
+        console.log("Je koos " + userInput.toLowerCase() + ". " + departments[userInput].description);
+        break;
+    default:
+        console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");
+        break;
+}
+
+console.log (userInput+ "  is een leuke afdeling om te werken. Er werken op dit moment " + departments[userInput].numberOfEmployees + " medewerkers.");
+
+const department = prompt("Je koos " + userInput + ". Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in. \n 0 = " + departments[userInput].jobs[0].title + "\n 1 = " + departments[userInput].jobs[1].title + "\n 2 = " + departments[userInput].jobs[2].title + "\n 3 = " + departments[userInput].jobs[3].title);
+switch (department) {
+    case "0":
+        console.log("Je koos " + departments[userInput].jobs[0].title + ". Een uitdagende rol! " + departments[userInput].jobs[0].description);
+        break;
+    case "1":
+        console.log("Je koos " + departments[userInput].jobs[1].title + ". Een uitdagende rol! " + departments[userInput].jobs[1].description);
+        break;
+    case "2":
+        console.log("Je koos " + departments[userInput].jobs[2].title + ". Een uitdagende rol! " + departments[userInput].jobs[2].description);
+        break;
+    case "3":
+        console.log("Je koos " + departments[userInput].jobs[3].title + ". Een uitdagende rol! " + departments[userInput].jobs[3].description);
+        break;
+    default:
+        console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");
+        break;
+}
```

Comment:
Je zou dit nog iets kunnen versimpelen
```suggestion
const index = Number(department);
if (department == "" || isNaN(index)) console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");

console.log("Je koos " + departments[userInput].jobs[index].title + ". Een uitdagende rol! " + departments[userInput].jobs[index].description);
```